pandas/io/parsers.py

```diff
@@ -2161,6 +2167,52 @@ def _buffered_line(self):
         else:
             return self._next_line()
 
+    def _check_for_bom(self, first_row):
+        """
+        Checks whether the file begins with the BOM character.
+        If it does, remove it. In addition, if there is quoting
+        subsequent to the BOM, remove it as well because it
+        technically takes place at the beginning of the name,
+        not the middle of it.
+        """
```

Comment:
"In addition, if there is quoting in the field subsequent to the BOM, remove it as well because it technically takes place at the beginning of the name..."